sriovnet_switchdev_test.go

```diff
@@ -63,8 +128,11 @@ func TestGetUplinkRepresentorErrorMissingSwID(t *testing.T) {
 func TestGetUplinkRepresentorErrorEmptySwID(t *testing.T) {
 	var testErr error
 	vfPciAddress := "0000:03:00.4"
+	uplinkRep := &repContext{"eth0", "p0", ""}
```

Comment:
if switch ID is empty we dont expect to have `phys_port_name` for uplink so perhaps set it to ""

Reply:
changed